### DISCUSSION TITLE
Battery won´t show up

### DISCUSSION THREAD

**Discussion by tober1** [wlcrs/huawei_solar#205] [Setting up the integration]
Hi,

Have successfully added my two inverters thru the Dongle/FE. The PowerMeter shows up automatically but not the batteries.
Should i also added the Mod-bus address for the battery's in the initial setup or have i missed something else?

I also se that my battery is lying on "battery_2" can that be the problem? I recently moved one battery pack from this inverter.

"slave_0": {
      "model_name": "SUN2000-5KTL-M1\u0000GIN300-5KTL-M1",
      "pv_string_count": 2,
      "has_optimizers": 0,
      "battery_1_type": 0,
      "battery_2_type": 2,
      "power_meter_type": 1

**Comment by wlcrs** [comment]
The code expects a battery to be present on battery 1 instead of battery 2. Any chance that you can just move over your battery to the correct connectors?

**Reply by tober1** [reply]
I can contact Huawei support an see if that is possible, else i probably need to reset the hole system.
